public/index.html

```diff
@@ -1,12 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    
-    <title>React App</title>
-  </head>
-  <body>
-    <div id="root"></div>
-  </body>
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" href="styles.css">
+  <title>Git App</title>
+</head>
+
+<body>
+  <div class="wrapper">
+    <div class="search">
+      <input class="search_input">
+      <button class="search_button">search</button>
+    </div>
+    <div class="items"></div>
```

Comment:
"items" is veeeeeeeeery common. items of what? users? flowers? pictures? be more concrete - `users-list` e.g.